arbitrum-docs/node-running/reference/software-releases.md

```diff
@@ -0,0 +1,58 @@
+---
+title: 'ArbOS software releases'
+sidebar_label: 'ArbOS software releases'
+author: dlee
+---
+
+:::info
+This page is meant to give a high level summary of node software releases, with particular emphasis on releases that bump the ArbOS version. To find information on all releases of Nitro, including the latest release, and accompanying changelogs, please visit the [Nitro Github Repository](https://github.com/OffchainLabs/nitro/releases).
+
+:::
+
+The name of the node software that is used to power Arbitrum is called Nitro. While Nitro releases can get shipped as frequently as multiple times per week, there are a few Nitro releases that carry an upgrade to [ArbOS](https://forum.arbitrum.foundation/t/arbitrum-arbos-upgrades/19695) - the Layer 2 EVM hypervisor that facilitates the execution environment of an Arbitrum chain. Releases of ArbOS can be considered as Arbitrum's equivalent to a hard fork and are special because they could alter a node's ability to produce valid Arbitrum blocks. Due to the impact that an ArbOS upgrade has on an Arbitrum chain, ArbOS upgrades are **mandatory to pick up** as a validator of a public Arbitrum chain. Note that every Nitro release is backwards compatible (i.e. supports all previous versions of ArbOS), which means your node will need to be at a Nitro version equal to or greater than the Nitro version that supports a given ArbOS release.
```

Comment:
```suggestion
Arbitrum chains are powered by Arbitrum nodes running the Nitro software stack. The Nitro software stack includes [ArbOS](https://forum.arbitrum.foundation/t/arbitrum-arbos-upgrades/19695), the Layer 2 EVM hypervisor that facilitates the execution environment of an Arbitrum chain.

Although Nitro is updated regularly, only a subset of Nitro releases carry ArbOS upgrades. These releases are significant because ArbOS upgrades are Arbitrum's equivalent to a hard fork - they can alter a node's ability to produce valid Arbitrum blocks. This is why validator nodes supporting a public Arbitrum chain (One, Nova) **must update Nitro** whenever a new ArbOS version is released.

Note that every Nitro release is backwards compatible. In other words, the latest version of Nitro will support all previous ArbOS releases. This means that your validator's Nitro version must be greater than or equal to the version that includes the latest ArbOS upgrade.
```

Minor edits for clarity and skimmability.